husky.config.js

```diff
@@ -1,6 +1,6 @@
 module.exports = {
   hooks: {
-    'pre-commit': 'lint-staged && node ./scripts/yarn-lock-scan.js',
+    'pre-commit': 'lint-staged',
```

Comment:
The reason we added yarn lock scan here is because we wanted to always scan it no matter what. But I think it's a bit too much indeed. We should rebase to latest master though, where lint-staged is handled in lintstagedrc file (has to do with package.json formatter I added last week)

Reply:
Yeah, I thought it was a bit too much, but I think it's better to be safe than sorry, since you could still circumvent  alltogether with `-n`

Can drop this commit again...